samples/README.md

```diff
@@ -843,3 +843,9 @@ must provide privileges for this sample to connect, publish, and receive.
 }
 </pre>
 </details>
+
+**Note:** There are some formatting and data requirements for custom metrics in Device Defender, which are listed below. If you input data that does not fit into the requirements below, it will be sent in the JSON packet like expected, but Device Defender will not show/retain the data! If you are trying to send data and not seeing it show up on Device Defender, please double check that your data fits into the requiements below.
```

Comment:
This reads a little funny to me. I kinda tried editing it a bit for clarity but it's entirely possible I'm the one that's just confused =P Either way, the last "requirements" is misspelled so we should at least fix that =]
**Note:** The formatting and data requirements for custom metrics in Device Defender are listed below. If you input data that does not follow the requirements, it will be sent in the JSON packet but Device Defender will not show/retain the data! If your data is not showing up in Device Defender, please double check that your data fits into the requirements below.